src/app/LayoutCommon.astro

```diff
@@ -17,22 +18,63 @@ envKeys.forEach((item) => {
 const sessionToken = getFromCookies(Astro.cookies, 'session');
 const refreshToken = getFromCookies(Astro.cookies, 'refresh');
 
+const allowedDomains = "'self' https://www.ayanworks.com https://qaapi.credebl.id https://devapi.credebl.id https://fonts.googleapis.com https://avatars.githubusercontent.com"
+
+// Astro.response.headers.set('Content-Security-Policy',"script-src 'self'; frame-ancestors 'self'");
+Astro.response.headers.set('X-Frame-Options', "SAMEORIGIN");
+Astro.response.headers.set('X-CREDEBL',"SSI");
```

Comment:
Is this line still needed? I think we can remove it.

Reply:
Removed the unused code.